common/app/model/EmailNewsletters.scala

```diff
@@ -1,409 +1,448 @@
 package model
 
-import controllers.ListIds
-
 case class EmailNewsletters(subscriptions: List[EmailNewsletter])
 case class EmailNewsletter(
   name: String,
   theme: String,
   teaser: String,
   description: String,
   frequency: String,
-  listId: String,
-  subscribedTo: (Iterable[String]) => Boolean,
-  subheading: Option[String],
-  tone: Option[String],
-  imageFilename: Option[String],
-  signupPage: Option[String],
-  exampleUrl: Option[String]
-)
-
-object EmailNewsletter {
-  def apply(emailSubscription: EmailNewsletter) = emailSubscription
-
-  def apply(
-    name: String,
-    theme: String,
-    teaser: String,
-    description: String,
-    frequency: String,
-    listId: String,
-    aliases: List[String] = List.empty,
-    subheading: Option[String] = None,
-    tone: Option[String] = None,
-    imageFilename: Option[String] = None,
-    signupPage: Option[String] = None,
-    exampleUrl: Option[String] = None
-  ): EmailNewsletter = EmailNewsletter(
-    name,
-    theme,
-    teaser,
-    description,
-    frequency,
-    listId,
-    subscribedTo = { subscriptions => subscriptions.exists((listId :: aliases).contains(_)) },
-    subheading,
-    tone,
-    imageFilename,
-    signupPage,
-    exampleUrl
-  )
+  listId: Int,
```

Comment:
Why model listId as an Int rather than a String? I did a double take on what subscribedTo is doing because of the different types.